client/state/sharing/publicize/actions.js

```diff
@@ -43,17 +43,33 @@ export function sharePost( siteId, postId, skippedConnections, message ) {
 		} );
 
 		return new Promise( ( resolve ) => {
```

Comment:
The `Promise` constructor can be removed. It's equivalent to just returning the result of `wpcom.req.post().then().catch()` if neither handler returns anything.

Reply:
Cleaned up in https://github.com/Automattic/wp-calypso/pull/57644/commits/78305a4bb7ece1c534d11d5eb4955c9a692efe9b.